src/main/java/appeng/me/GridNode.java

```diff
@@ -455,10 +456,13 @@ else if( isValidConnection )
 						// construct a new connection between these two nodes.
 						try
 						{
-							new GridConnection( node, this, f.getOpposite() );
+							GridConnection.create( node, this, f.getOpposite() );
 						}
 						catch( final FailedConnectionException e )
 						{
+							e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1148)